lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -160,6 +160,7 @@ def warn_about_bare_error
                         "will match when Ruby raises a `NoMethodError`, `NameError` or " \
                         "`ArgumentError`, potentially allowing the expectation to pass " \
                         "without even executing the method you are intending to call. " \
+                        "Actual error raised was #{description_of(@actual_error)}. "\
```

Comment:
that works for me....


```
  1) expect { ... }.to raise_error issues a warning that does not include current error when its not present
     Failure/Error: RSpec::Support.notify_failure(RSpec::Expectations::ExpectationNotMetError.new message)
       expected "WARNING: Using the `raise_error` matcher without providing a specific error or message risks false positives, since `raise_error` will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`, potentially allowing the expectation to pass without even executing the method you are intending to call. Actual error raised was nil. Instead consider providing a specific error class or message. This message can be supressed by setting: `RSpec::Expectations.configuration.warn_about_potential_false_positives = false`. Called from /Users/xavier/Code/ex/rspec-expectations-isolate/spec/rspec/matchers/built_in/raise_error_spec.rb:26:in `block (2 levels) in <top (required)>'." not to match /Actual error raised was/
       Diff:
       @@ -1,2 +1,2 @@
       -/Actual error raised was/
       +"WARNING: Using the `raise_error` matcher without providing a specific error or message risks false positives, since `raise_error` will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`, potentially allowing the expectation to pass without even executing the method you are intending to call. Actual error raised was nil. Instead consider providing a specific error class or message. This message can be supressed by setting: `RSpec::Expectations.configuration.warn_about_potential_false_positives = false`. Called from /Users/xavier/Code/ex/rspec-expectations-isolate/spec/rspec/matchers/built_in/raise_error_spec.rb:26:in `block (2 levels) in <top (required)>'."

     # /Users/xavier/.gem/ruby/2.2.2/bundler/gems/rspec-support-81e33d4ed3d9/lib/rspec/support.rb:87:in `block in <module:Support>'
```